package.json

```diff
@@ -16,7 +16,7 @@
     "build": "webpack --mode production",
     "build:dev": "webpack --mode development",
     "build:watch": "npm run build:dev -- --watch",
-    "clean": "rm -rf _site tests/_artifacts",
+    "clean": "rimraf _site tests/_artifacts",
```

Comment:
Seems that you haven't properly rebased. Please, rebase with current master.

Reply:
I'm not sure what you mean. 
What is the issue here?

Reply:
This change has been added in a different PR, you need to merge `master` with this branch.

Reply:
```bash
git checkout master
git pull origin master
git checkout issue-16-2
git merge master
```

Reply:
That's what I did, I don't remember copying this manually.
I did
Git checkout master
Git pull
Git checkout issue-16-2
Git rebase -i master

I can rebase again.

Reply:
Please, rebase again, but try to not open another PR because we loss a lot of context going through PRs.

Reply:
Ok

Reply:
I've merged from master. it brought the changes related to #138, as expected, and nothing else.